fs/fs.go

```diff
@@ -1,7 +1,7 @@
-package gps
+package fs
```

Comment:
I think I'd rather make this `internal/fs`, at least for now - these funcs aren't really intended to be a part of the public surface area.